File: src/Eredua/Filma.java
package Eredua;

public class Filma {
    private int id;
    private String izena;
    private String deskribapena;
    private double puntuazioa;
    private int alokairuKop;
    private int iruzkinKop;
    
    public Filma(String pIzena, String pDeskribapena){
        this.id = 0; //ponerle un id distinto a cada film
        this.izena = pIzena;
        this.deskribapena = pDeskribapena;
        this.puntuazioa = 0;
        this.alokairuKop = 0;
        this.iruzkinKop = 0;
    }
}

File: src/Eredua/Filma.py
class Filma:
    def Filma(self, id, izena, deskribapena, puntuazioa, alokairuKop, iruzkinKop):
        self.id = id
        self.izena = izena
        self.deskribapena = deskribapena
        self.puntuazioa = puntuazioa
        self.alokairuKop = alokairuKop
        self.iruzkinKop = iruzkinKop

    def inf_lortu(self):
        return f"{self.id} - {self.izena} - {self.deskribapena} - {self.puntuazioa} - {self.alokairuKop} - {self.iruzkinKop}"
    
    